api.planx.uk/modules/send/idox/nexus.ts

```diff
@@ -413,7 +411,7 @@ const createUniformApplicationAuditRecord = async ({
       destination: localAuthority,
       response: submissionDetails,
       payload,
-      xml,
+      xml: "ODP Schema",
```

Comment:
In the future we could think about a new column to snapshot the schema payload or an entirely separate audit table - but for this stage of testing we can simply make a static note here and then do any schema troubleshooting using our admin endpoitns (eg `admin/session/:sessionId/digital-planning-application` where PlanX session ID = Idox submission reference)